### PR TITLE
Add support for managing a pod's configuration as a pod owner

### LINKED ISSUE #128: Manage Pod -- Admin Panel
As a Pod Admin / Owner we want to be able to make "configuration" changes on a per-Pod basis that are easy to do without having to edit a configuration file or environment variables, etc. This is especially useful for the [Twt.social](https://twt.social) fully-managed hosted pods offering we run.

As a basic minimum there should be an `/_admin` view that allows a Pod Owner (_identified as `Config.AdminUser`_) to:

- Upload a custom logo and favicon for their pod, replacing the default.
- Update the pod's name and description.
- Enable/Disable open registration
- Enable/Disable open profiles
- Change the maximum twt length

### PR DESCRIPTION
**Pull request by shahxeb**
Closes #128

### CODE REVIEW

File: internal/config.go [hunk 1]
```diff
@@ -1,57 +1,65 @@
...
+	"io"
 	"io/ioutil"
 	"math/rand"
 	"net/url"
 	"os"
 	"regexp"
 	"strings"
 	"time"
+
+	"github.com/apex/log"
```
Comment (prologic) [inline internal/config.go:14]: Please use logrus :)

**Review by prologic — CHANGES_REQUESTED** [review]
Looks mostly okay to me. Basically what we expect! 👍 

Please fix the conflicts so I can test this!

**Comment by prologic** [comment]
Fixed a few issues in this PR that were the result of bad merge I think.

There is still one outstanding problem; the "configuration" changes don't persist after a restart. I can see they are written to a config file, but we don't seem to load that config file back at all.

File: internal/config.go [hunk 2]
```diff
@@ -1,57 +1,91 @@
...
+)
+
+var (
+	ErrConfigPathMissing = errors.New("error: config file missing")
 )
 
+// Settings contains Pod Settings that can be customised via the Web UI
+type Settings struct {
+	Name        string `yaml:"pod_name"`
+	Description string `yaml:"pod_descriptio"`
```
Comment (aaadonai) [inline internal/config.go:25]: missing the last n on description.

Comment (prologic) [reply internal/config.go:25]: Thanks!

**Comment by prologic** [comment]
I just need to rebase this and resolve the conflicts (_its a bit of a mess unfortunately_) and we can merge this!

**Review by prologic — APPROVED** [review]

**Pull request merged**